front/cypress/e2e/survey_builder/create_and_fill_survey.cy.ts

```diff
@@ -2,7 +2,15 @@ import { snakeCase } from 'lodash-es';
 import { randomString, randomEmail } from '../../support/commands';
 import moment = require('moment');
 
-describe.skip('Survey builder', () => {
+const waitForCustomFormFields = () => {
```

Comment:
All tests in this file were disabled, due to flakyness that was caused because we need to wait a little bit before we can click on a custom field input type to add it to the form (without waiting, clicking doesn't result in adding the field to the form).